lib/package.gd

```diff
@@ -691,6 +691,31 @@ DeclareGlobalFunction( "LoadPackageDocumentation" );
 ##
 ##  <Description>
 ##  loads the &GAP; package with name <A>name</A>.
+## <P/>
+##  As an example, the following loads the &GAP; subgroups package
```

Comment:
> Seems the 'gapdoc' package cannot be accessed by the GAP build in the repo.

Yes. The easiest way to get it working is perhaps to put a symlink `pkg` in the GAP root directory pointing to the `pkg` directory from the latest GAP installation.